crates/scheduler/src/call/button.rs

```diff
@@ -42,30 +40,35 @@ fn count<B: Board>(call: SchedulerCall<B, api::count::Sig>) {
 
 #[cfg(feature = "board-api-button")]
 fn register<B: Board>(mut call: SchedulerCall<B, api::register::Sig>) {
+    use wasefire_error::{Code, Error};
```

Comment:
Let's add this at the top instead (like you did in the other modules):

```rust
#[cfg(feature = "board-api-button")]
use wasefire_error::{Code, Error};
```

I think rust-analyzer adds imports to the function when the function is feature-gated. That's sometimes the right choice, but not always (e.g. when multiple functions with the same feature-gate need the same imports).

Reply:
Done.